src/plugins/telemetry/public/plugin.ts

```diff
@@ -153,6 +155,11 @@ export class TelemetryPlugin implements Plugin<TelemetryPluginSetup, TelemetryPl
       telemetryConstants = getTelemetryConstants(docLinks);
     });
 
+    analytics.registerShipper(ElasticV3BrowserShipper, {
+      channelName: 'kibana-browser',
+      version: currentKibanaVersion,
+    });
```

Comment:
Optional: can add a test in `src/plugins/telemetry/public/plugin.test.ts` to make sure that this call is properly performed.